### DISCUSSION TITLE
Milestone 3 (v3.0.0)

### DISCUSSION THREAD

**Discussion by SoloByte** [DaveGreen-Games/ShapeEngine#23] [Announcements]
There is nothing here yet.

**Comment by guisantos** [comment]
Maybe add unit tests? I can help with that if you give me a headstart where you wanna put them and what you'd like to test

**Reply by SoloByte** [reply]
I do not really need them right now. 

I want to start using ShapeEngine for my next game to find out how good/bad ShapeEngine works for making complex games right now. I already have been delayed for over a year ^^ 

After all I want to know if it works for production and not just on paper or simple examples.

**Comment by SoloByte** [comment]
I renamed the milestone to 3.0 because the 1.1 release became so big that I decided it will be the 2.0 release instead.
